src/api-reference/expense/attendees/v4.attendees.markdown

```diff
@@ -0,0 +1,541 @@
+---
+title: Attendees v4
+layout: reference
+---
+
+{% include prerelease.html %}
```

Comment:
should we be including this pre release tag?

Reply:
This will be removed by the tech writers right before release.